### DISCUSSION TITLE
Proposal for the Development of Liquid Staked AKT (stkAKT) by pSTAKE

### DISCUSSION THREAD

**Discussion by bartjhv** [akash-network/community#508] [Integrations]
### Greetings, AKASH community!

We, at pSTAKE, are excited to present a proposal to develop Liquid Staked AKT (stkAKT) for Akash. This post aims to introduce pSTAKE’s Liquid Staking solution to the Akash community first, we have not communicated this elsewhere. Our goal is to enhance the utility and liquidity of AKT while maintaining the security and decentralisation of the AKASH Network. We look forward to your feedback and suggestions. 

### Background
pSTAKE is a multi-chain liquid staking protocol that unlocks liquidity for your staked assets using modules deployed on the Persistence chain. With pSTAKE, you can securely stake your Proof-of-Stake (PoS) assets to earn staking rewards, and receive underlying representative tokens (stkAssets) which can be used to explore additional yield opportunities across DeFi.

At present, pSTAKE supports Cosmos (ATOM), Binance chain (BNB), dYdX network (DYDX), Osmosis (OSMO) and Stargaze (STARS) with Persistence (XPRT) in the future.

### Proposal
We propose to develop stkAKT, a Liquid Staked version of the AKT token. This will enable AKT holders to stake their tokens while maintaining liquidity, allowing holders to participate in DeFi activities without sacrificing their staking rewards.

### Benefits
- **Enhanced Liquidity:** stkAKT will provide liquidity to staked AKT, enabling holders to participate in DeFi activities without unbonding their tokens.
- **Increased Utility:** The integration of stkAKT into DeFi protocols will create new use cases for AKT, potentially increasing its demand and value.
- **Network Security:** By incentivizing more holders to stake their AKT, we can enhance the security and decentralisation of the AKASH network.
- **More Yield:** stkAKT will be auto compounded to ensure maximum yield on your staked tokens.
- **Community Growth:** The introduction of stkAKT can attract new users and investors to the AKASH ecosystem, fostering community growth and development.
- **Superior Liquid Staking UX:** pSTAKE allows instant LST redemptions on its dApp directly. For example, users can instantly redeem stkAKT for AKT (paying a small fee of 1%) on pSTAKE. This works thanks to a novel mechanism that will match daily AKT deposits into the protocol with daily stkAKT withdrawal requests.

### Implementation
Technical Development: pSTAKE and Persistence Labs will manage the infrastructure and continuous support required to enable stkAKT. Details on the pSTAKE architecture can be found [here](https://docs.pstake.finance/). Note that ICA has not been initialised on Akash, pSTAKE would raise a PR to enable the module. This would only enable a few transaction types and should not impact the workings of Akash.

Ensuring the security of staked assets and liquidity provision is paramount. pSTAKE employs state-of-the-art security measures with smart contract audits and vulnerability assessments, to safeguard users' assets and transactions (with audits by Trail of Bits, Peckshield, Halborn and Certora).

Our team will formulate and implement effective business development strategies to promote the adoption of stkAKT. We will work together with Akash to establish partnerships and integrations (DEX’s, Lending, Vaults…). Additionally we can create educational content to communicate the benefits of (liquid) staking and liquidity provision. 

Initially we would request some community owned liquidity provision to bootstrap pools with AKT/stkAKT. This could be on Osmosis, Dexter, Astroport etc. pending on the community’s feedback. 

### Conclusion
We believe that the development of stkAKT will bring significant value to the AKASH community by enhancing liquidity, increasing utility, and strengthening network security. We are committed to working closely with your community to ensure the success of this initiative and look forward to your support and feedback.

Let us know what you think!
Bart Van der Voort - Head of Strategy

**Comment by ghost** [comment]
Hey there, thanks for starting a discussion about LS AKT, I totally get the benefits of LS 

About this line: "Initially we would request some community owned liquidity provision to bootstrap pools with AKT/stkAKT." 
How much will this cost Akash? in total

**Reply by bartjhv** [reply]
Hi Robert,

Thanks for the feedback! In order to have enough trading liquidity, we would recommend around $2M (~10% of the community pool). This would be split 50/50 in AKT and stkAKT. The liquidity ownership remains with the community. In fact, it will yield staking awards on the stkAKT portion (95% goes to stakers, 5% goes to pSTAKE protocol) & potentially trading fees. This should be a net positive for the AKT community pool.

Of course, this is just a suggestion, we're happy to get recommendations from the Akash community.

**Comment by piber-dev** [comment]
hello @bartjhv, i am wondering a few things before i can express my opinion on this proposal.

1) what risks do you think this could be relevant when implementing this? i am assuming that there would only be the possibility of larger risks at the smart contract side.
2) what yields has been seen from your integrations with other projects? and if i understand correctly, this is added on top of the staking rewards found on akash already? or perhaps you can only earn at one place at a time? akash already has a fairly large inflation rate of approximately 15%.
3) is this an opt-in solution where users can choose whether or not they want to take part of this DeFi solution? especially if there are risks attached, i hope that it's opt-in. some users also think that their taxes are hard enough as it is, is there a good solution for managing taxes with your service?

**Reply by bartjhv** [reply]
Hi @piber-dev 

To answer your questions: 

1.  There would be a protocol risk on the pSTAKE side, however our module has been [audited thoroughly](https://docs.pstake.finance/Security%20Audits/) so we are confident in its security. Other than this, there could be DeFi risk with integrations of other protocols.
2. Correct, yields would be on top of the staking rewards. This depends on the integrations, but you could LP stkAKT, use it as collateral for lending, integrate with vault products etc.
3. Yes, its completely opt-in. It just gives users the opportunity to retain staking rewards and the option to participate in DeFi or keep their staked assets liquid. 

Hope this clarifies!

**Comment by rodri-r** [comment]
So in your first paragraph you mention that you have not mentioned this anywhere else, however there are already some people talking about this in X/Twitter since March 11th.

Clearly you or someone in your team has communicated about this elsewhere and are not being completely transparent here.

In my opinion this could harm the network in many ways and it is not something that really benefits the network at the moment.

![stkakt.jpg](https://github.com/akash-network/community/assets/46242399/49e437ad-c960-44fb-94f4-b70f7e0717b9)

**Reply by rodri-r** [reply]
Here is a link to main blog post, which proves that you have mentioned this in other places besides this repo

https://blog.pstake.finance/2024/03/11/pstake-thesis-cosmos-liquid-staking-will-be-the-biggest-market-in-crypto/

**Reply by bartjhv** [reply]
Hi @rodri-r - thanks for the feedback! I'm not sure though if we're on the same page here. We have not announced officially launching stkAKT, but that doesn't mean community members have not talked (or requested) it. Akash has long been recognized as an interesting partner for Liquid Staking.

Your opinion that it would harm the network, do you mind elaborating on this? 

The development and distribution of stkAKT should actually boost the economic security and staking ratio of AKT while contributing to the further decentralisation of the network. pSTAKE Finance delegates to Akash validators, selected transparently and based on on-chain parameters like governance participation, voting, slashing, and more. Liquid staking AKT is also fully non-custodial.

**Reply by rodri-r** [reply]
Hello, well there is a mention about stkAKT in a graph in your blog post, which seems to me like a public announcement of this coming.

Akash is permisionless and decentralized, you could just launch stkAKT without requesting funds from community.

From what I have seen in other tokens/networks that have enabled liquid staking it has not caused token price to go up, people can just sell their tokens or lose them in other protocols, thus not having access to staking rewards anymore nor to governance. If this happens your protocol could also just sell remaining or liquid staked AKT to recover what you gave as stkAKT tokens.

Could be wrong, but in my opinion just having normal staking without liquid staking helps to protect the network a lot more, it still encourages holders to stake and does not cause as much selling pressure.

**Comment by bartjhv** [comment]
> Hello, well there is a mention about stkAKT in a graph in your blog post, which seems to me like a public announcement of this coming.

Ok sure, we did mention interest.
> 
> Akash is permisionless and decentralized, you could just launch stkAKT without requesting funds from community.

This is correct, but of course its preferable if the community is on board. The community funds would be to bootstrap initial liquidity on a DEX. Again, this is protocol owned liquidity. We have no ownership of this and it would actually yield returns for the community (staking rewards + trading fees).
> 
> From what I have seen in other tokens/networks that have enabled liquid staking it has not caused token price to go up, people can just sell their tokens or lose them in other protocols, thus not having access to staking rewards anymore nor to governance. If this happens your protocol could also just sell remaining or liquid staked AKT to recover what you gave as stkAKT tokens.

The goal shouldn't be for the price to go up. Its to give users more options in DeFi without forsaken their staking rewards. More yield basically.
> 
> Could be wrong, but in my opinion just having normal staking without liquid staking helps to protect the network a lot more, it still encourages holders to stake and does not cause as much selling pressure.

Your point about traditional staking providing more network protection is valid, as it encourages long-term holding and reduces selling pressure. However, liquid staked tokens (LSTs) offer the dual benefits of maintaining network security through staking while also providing liquidity to holders. Because of the liquidity it can actually increase participation in the staking process and attract more capital to the ecosystem. Time will tell, but we actually foresee a future where LSTs will surpass base assets and become the predominant asset for exchange and utility in DeFi.

**Reply by rodri-r** [reply]
Makes some sense too!
